### PR TITLE
OCPQE-22436 Adding y version upgrade jobs for OCP 4.17

### PR DESCRIPTION
**Pull request by JianLi-RH**
/cc @jianlinliu @shellyyang1989 @jiajliu @dis016 @evakhoni

### CODE REVIEW

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-4.17-upgrade-from-stable-4.16-aws-ipi-ingress-lbtype-nlb-arm-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-4.17-upgrade-from-stable-4.16-baremetal-compact-agent-ipv4-static-connected-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-stable-4.16-upgrade-from-stable-4.16-aws-ipi-proxy-sts-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-multi-stable-4.17-upgrade-from-stable-4.16-aws-ipi-ovn-ipsec-amd-mixarch-f14

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-stable-4.17-upgrade-from-stable-4.16-aws-c2s-ipi-disc-priv-fips-f14

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-stable-4.16-upgrade-from-stable-4.16-aws-ipi-proxy-sts-f28

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-4.17-upgrade-from-stable-4.16-aws-ipi-disc-priv-sts-f28

**Comment by mhanss** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-4.17-upgrade-from-stable-4.16-vsphere-agent-compact-fips-f28

**Comment by shellyyang1989** [comment]
@liangxia PTAL Thank you!

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml [hunk 1]
```diff
@@ -0,0 +1,661 @@
...
+  cron: 51 9 7 * *
+  steps:
+    cluster_profile: vsphere-dis-2
+    leases:
+    - env: VSPHERE_BASTION_LEASED_RESOURCE
+      resource_type: vsphere-connected-2-quota-slice
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-vsphere-ipi-proxy-workers-rhel8
+- as: vsphere-ipi-sdn-f28
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml:606]: @zhaozhanqi do we still support the sdn cluster upgrade in 4.17 cycle?

Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml:606]: @zhaozhanqi   Do we support upgrade from 4.16 to 4.17 for SDN ?

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml:606]: `SDN has been removed from >= 4.15 fresh install`. @zhaozhanqi please confirm

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml:606]: `SDN has been removed from >= 4.15 fresh install`. @zhaozhanqi please confirm

Comment (zhaozhanqi) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml:606]: no, sdn cannot upgrade to 4.17

**Pull request merged**